akka-actor/src/main/scala/akka/util/ByteIterator.scala

```diff
@@ -443,9 +444,9 @@ abstract class ByteIterator extends BufferedIterator[Byte] {
     if (found) index else -1
   }
 
-  def indexOf(elem: Byte): Int = indexWhere { _ == elem }
+  def indexOf(elem: Byte): Int = indexWhere { (_: Byte) == elem }
 
-  override def indexOf[B >: Byte](elem: B): Int = indexWhere { _ == elem }
+  override def indexOf[B >: Byte](elem: B): Int = indexWhere { (_: Byte) == elem }
```

Comment:
I’m fine with that. Being phased out anyway

Reply:
Yeah, more curious than a complaining.

Reply:
@johanandren to satisfy your curiousity, this is because the indexOf function in 2.13 takes a default parameter.

Scala 2.12.6
https://github.com/scala/scala/blob/d7ae7a8b9668d52699d1e1c5066a22ec89a1d379/src/library/scala/collection/Iterator.scala#L1009-L1033

Scala 2.13.
https://github.com/scala/scala/blob/a0026305125dcc814097b1fbb798fe925ca9f81f/src/library/scala/collection/Iterator.scala#L331-L339

Morale of the story: Don't use default parameters. See https://github.com/scala/bug/issues/10973